demo/demo-worker.js

```diff
@@ -130,6 +132,12 @@ maybeReset().then(async () => {
     sqlite3.vfs_register(vfs, true);
   }
 
+  if(buildName.startsWith('mc-')) {
```

Comment:
Is there another way we can determine this is needed, without relying on the file name convention? For example checking if `sqlite3mc_vfs_create` is defined?

Reply:
I see - I missed that this is just the demo.

Would the web SDK contain similar logic to this?